src/input/Slider.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ChangeEvent } from "react";
+
+import styles from "./Slider.module.css";
+
+interface SliderProps {
+  value?: number;
+  onChange: (value: number) => void;
+  defaultValue?: number;
+  step?: number;
+  min?: number;
+  max?: number;
+  sliderTrackOpacity?: number;
```

Comment:
This seems not used? Was it supposed to be used to inform the color of the slider?

Reply:
This was used for the opacity, but hugo might have exchanged that for the implementation inside the slider.tsx as it gave a better endresult.